Docs/source/Model.rst

```diff
@@ -183,34 +184,32 @@ The mixture-averaged transport coefficients discussed above (:math:`\mu`, :math:
 
 The following choices are currently implemented in `PeleLM`
 
-* The viscosity, :math:`\mu`, is estimated based <something>
+* The viscosity, :math:`\mu`, is estimated based on one step of the conjugate gradient method, using temperature dependent ratios of collisions integralsi (EGZE3).
```

Comment:
I suppose there is a typo in "integralsi" here?

Reply:
thanks done